### ISSUE TITLE
Selecting users to be added to a circle should show their mail address

### ISSUE THREAD

**Issue by Volker-K** [nextcloud/circles#1047]
The larger the installation, the greater the likelihood that duplicate names will occur. When sharing files and folders with users, their mail address is displayed in the selection.
When adding users to a circle, this should also be done so that in case of name duplicates there is a chance to identify the correct user.

**Comment by ArtificialOwl** [comment]
@skjnldsv i do not think this is directly related to Circles, do you think this is implementable and where should I forward this request ?

**Comment by Volker-K** [comment]
> @skjnldsv i do not think this is directly related to Circles, do you think this is implementable and where should I forward this request ?

Well, when I am sharing via Files App I see the mail address, when I add a user to a circle in Contacts-/Circles-App I don't. I can't determine ad hoc where the subroutine is located, but I think it should be Circles App. 
If Circes uses a subroutine that does not display/deliver the mail address this shold be forwarded as request to where this routine is located ;-)

**Comment by DerDreschner** [comment]
Another thumbs up from my side for this issue as well. It's pretty confusing when you have multiple users with the same display name as there is no way do differentiate in case there is no profile picture available for any of them.

<img width="651" height="528" alt="Image" src="https://github.com/user-attachments/assets/69a25e7e-bc75-498c-bdc9-05369a061c36" />
